### ISSUE TITLE
Add possibility to scan ConfigMaps in addition to Secrets

### ISSUE THREAD

**Issue by nweisenauer-sap** [enix/x509-certificate-exporter#208]
Hey,

we are successfully using this tool to keep track of certificates inside K8s secrets and files on the nodes.
However, some certificates are stored inside K8s ConfigMaps. Is it possible to scan for certificates inside ConfigMaps?

Cheers,
 Nico

**Comment by Zhaikuku** [comment]
Yes, for example, the (kubectl get cm -A | grep istio-ca-root-cert) configuration is obtained, which is a required configuration for istio v1.10

**Comment by npdgm** [comment]
Sorry for the silence on this issue. This project was awaiting a massive overhaul.
Please give a try to version 4, support for ConfigMaps was greatly improved.
